ekf/ekflib.c

```diff
@@ -261,24 +280,45 @@ static void *ekf_thread(void *arg)
 	time_t imuTime;
 	time_t loopStep, updateStep;
 	update_engine_t *currUpdate;
+	bool dataAcqSucc;
```

Comment:
Slight modification for more appropriate name ;)
```suggestion
	bool dataAcqOk;
```
On the other hand `ekf_common.run` may be used for this purpose: just set this variable to false in `*Acq()` ladder and then before math part there is check if we still want to run the loop.